lbz/exceptions.py

```diff
@@ -13,13 +13,14 @@
 class LambdaFWException(Exception):
     """
     Standarised for AWS Lambda exception class.
+    Log Levels: CRITICAL, ERROR, WARNING, INFO, DEBUG
     """
 
     message = HTTPStatus.INTERNAL_SERVER_ERROR.description
     status_code = HTTPStatus.INTERNAL_SERVER_ERROR.value
 
-    def __init__(self, message: str = ""):
-        super().__init__()
+    def __init__(self, *args, message: str = None):
```

Comment:
That change needs to be checked carefully during updating Lambdalizator in our services 🤔 

I can imagine, we are passing the message as a positional argument, and problems related to that won't be caught automatically (at least not everywhere).

Reply:
fixed - got deep with base exception class